### PR TITLE
feat: contacts and pass code feature flags

### PR DESCRIPTION
**Pull request by yuriysych**

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -7,6 +7,28 @@
...
+      "type": "boolean",
+      "title": "Contacts tab",
+      "default": true,
+      "description": "if true = contacts tab is enable for users"
+    },
+    "isHidingContentInTaskManagerEnabled": {
+      "type": "boolean",
+      "title": "Hide app content in task manager",
+      "default": true,
+      "description": "if true = app content will be hidden in task manager"
```
Comment (a23sokolov) [inline docs/feature-flags.schema.json]: also: disabled screenshot, video record program cant catch UI

Comment (yuriysych) [reply docs/feature-flags.schema.json]: поправил

File: docs/feature-flags.schema.json [hunk 2]
```diff
@@ -7,6 +7,28 @@
       "default": true,
       "description": "if true = discover tab is enable for users"
     },
+    "isContactsActive": {
```
Comment (nkt) [inline docs/feature-flags.schema.json]: rename to `tabs. isContactsActive `

File: docs/feature-flags.schema.json [hunk 3]
```diff
@@ -7,6 +7,28 @@
...
+      "default": true,
+      "description": "if true = contacts tab is enable for users"
+    },
+    "isHidingContentInTaskManagerEnabled": {
+      "type": "boolean",
+      "title": "Hide app content in task manager",
+      "default": true,
+      "description": "if true = app content will be hidden in task manager, disabled screenshot, video record program cant catch UI"
+    },
+    "failedAttemptsTillBlocking": {
```
Comment (nkt) [inline docs/feature-flags.schema.json]: rename to `passcode.failedAttemptsTillBlocking `

File: docs/feature-flags.schema.json [hunk 4]
```diff
@@ -7,6 +7,28 @@
...
+      "type": "integer",
+      "title": "Wrong unlocks count before disabling pass code input",
+      "default": 3
+    },
+    "failedAttemptsTimeoutMillis": {
```
Comment (nkt) [inline docs/feature-flags.schema.json]: rename to `passcode.failedAttemptsTimeoutMillis`

File: docs/feature-flags.schema.json (hunk 3 above)
Comment (yuriysych) [reply docs/feature-flags.schema.json]: сделал

File: docs/feature-flags.schema.json (hunk 4 above)
Comment (yuriysych) [reply docs/feature-flags.schema.json]: сделал

File: docs/feature-flags.schema.json (hunk 2 above)
Comment (yuriysych) [reply docs/feature-flags.schema.json]: сделал

**Pull request merged**